src/test/java/org/grizz/web/api/PlayerControllerTest.java

```diff
@@ -63,6 +67,48 @@ public void shouldReturnUserWhenChangingPassword() throws Exception {
                 .andExpect(jsonPath("$.login").value(LOGIN));
     }
 
+    @Test
+    public void shouldCreateNewUserWhenGivenLoginAndPassword() throws Exception {
+        Gson gson = new Gson();
+        NewUserCreateRequest request = NewUserCreateRequest.builder()
+                .login(LOGIN)
+                .password(NEW_PASSWORD)
+                .build();
+        String json = gson.toJson(request, NewUserCreateRequest.class);
+        mockMvc.perform(post("/players")
+                .contentType(MediaType.APPLICATION_JSON_UTF8)
+                .content(json))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    public void shouldThrowExceptionWhenLoginIsTooShort() throws Exception {
+        Gson gson = new Gson();
+        NewUserCreateRequest request = NewUserCreateRequest.builder()
+                .login(SHORT_LOGIN)
+                .password(NEW_PASSWORD)
+                .build();
+        String json = gson.toJson(request, NewUserCreateRequest.class);
+        mockMvc.perform(post("/players")
+                .contentType(MediaType.APPLICATION_JSON_UTF8)
+                .content(json))
+                .andExpect(status().is4xxClientError());
```

Comment:
Also the exact 4xx status code is important here. Now it could be 404 Not Found or even 403 Unauthorized. We would never know